pyproject.toml

```diff
@@ -36,7 +36,7 @@ classifiers = [
 ]
 dependencies = [
     "async_lru>=1.0.0",
-    "httpx>=0.25.0",
+    "httpx~=0.28.0",
```

Comment:
As a potentially-security-related package, having a pin of `>=0.28.0,<0.29.0` seems like we're asking for a rushed future release.